docs/proposals/optional_application_loging.md

```diff
@@ -0,0 +1,144 @@
+Optional Application Logging
+============================
+
+The EFK aggregated logging stack collects application logs from STDOUT of each 
+container managed by OpenShift using fluentd. Currently, only way of opting out
```

Comment:
I think this is a little misleading talking about 'opting out' here. Would we want to say something about how there is no granular control currently... logs are all or nothing based on what nodes fluentd is deployed to (which flows into the next sentence).